micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/index.js

```
@@ -56,23 +60,39 @@ const TopBarSideBar = ({
-          t={t}
-          CITIZEN={CITIZEN}
-          isSidebarOpen={isSidebarOpen}
-          toggleSidebar={toggleSidebar}
-          handleLogout={handleLogout}
-          mobileView={mobileView}
-          userDetails={userDetails}
-          linkData={linkData}
-          islinkDataLoading={islinkDataLoading}
-        />
-      )}
+      {showDialog && <LogoutDialog onSelect={handleOnSubmit} onCancel={handleOnCancel} onDismiss={handleOnCancel}></LogoutDialog>}
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)